readthedocs/oauth/models.py

```diff
@@ -62,6 +62,10 @@ def is_admin(self):
             return full_json['permissions']['admin']
         return False
 
+    def is_private(self):
+        full_json = eval(self.json)
```

Comment:
I'm dubious of `eval` here, it could possibly open up to security issues as this is a weak serialization. JSON would be a better format for this field, but outside the scope of this PR. Maybe this would be a good case for `JSONField` though.

I'm not horribly worried about this as an attack vector as far as this PR is concerned, as the input should be fairly sane coming from Github's API. Better serialization would be an improvement to follow up on though.